core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -2370,6 +2370,47 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                                                 &array_elem);
                         HANDLE_OP_END();
                     }
+#if WASM_ENABLE_GC_BINARYEN != 0
+                    case WASM_OP_ARRAY_COPY:
+                    {
+                        uint32 dst_offset, src_offset, len, src_type_index;
+                        WASMArrayObjectRef src_obj, dst_obj;
+
+                        read_leb_uint32(frame_ip, frame_ip_end, type_index);
+                        read_leb_uint32(frame_ip, frame_ip_end, src_type_index);
+
+                        dst_obj = POP_REF();
+                        dst_offset = POP_I32();
+                        src_obj = POP_REF();
+                        src_offset = POP_I32();
+                        len = POP_I32();
+
+                        if (!src_obj || !dst_obj) {
+                            wasm_set_exception(module, "null array object");
+                            goto got_exception;
+                        }
+
+                        if (len > 0) {
+                            if (((uint64)dst_offset + (uint64)len >= UINT32_MAX)
```

Comment:
`=` should be allowed. And how about checking it according to SDL rule:
`len > UINT32_MAX - dst_offset`
We used it in:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/common/wasm_memory.c#L277-L278